src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -296,7 +296,8 @@ private Matcher<JobModel>[] getJobMatchers(List<JobModel> jobModels) {
   private String runFlight(
       String description, Class<? extends Flight> clazz, AuthenticatedUserRequest testUser) {
     String jobId = jobService.newJob(description, clazz, null, testUser).submit();
-    jobService.waitForJob(jobId);
+    // Poll repeatedly with no breaks: we expect the job to complete quickly.
```

Comment:
I did not refactor this test suite to the extent that would be desirable: it still runs in JUnit 4 and would be more accurately classified as a connected test.  But it was by far and away the longest running unit test suite at close to 3 minutes, so I wanted to take a closer look.

By altering the Stairway polling to eliminate its sleep in between checking that test dummy flights complete, we dramatically drive down test suite runtime: individual tests that previously took over a minute to run excluding initialization now take under a second.

Ironically, this had the biggest impact on test runtimes vs. the more extensive refactors I pursued!